bundles/net.tourbook.common/src/net/tourbook/common/UI.java

```diff
@@ -153,6 +153,7 @@
    public static final String       SYMBOL_AVERAGE_WITH_SPACE     = "\u00f8 ";  //$NON-NLS-1$
    public static final String       SYMBOL_BOX                    = "\u25a0";   //$NON-NLS-1$
    public static final String       SYMBOL_BULLET                 = "\u2022";   //$NON-NLS-1$
+   public static final String       SYMBOL_CHECK                  = "\u2705";   //$NON-NLS-1$
```

Comment:
A useful name for a symbol is the name which is displayed in this dialog, this is the way I name it, however not since the beginning

![mt-symbol-name](https://user-images.githubusercontent.com/1283445/109310425-cb5e5b80-7844-11eb-9d94-367208ee7bef.png)

Reply:
Good suggestion, I will keep that in mind.
I fixed this in this [PR](https://github.com/wolfgang-ch/mytourbook/pull/308)